crates/networking/discv5/src/discovery.rs

```diff
@@ -101,7 +101,17 @@ impl Discovery {
             }
             if let Err(err) = discv5.add_enr(enr) {
                 error!("Failed to add bootnode to Discv5 {err:?}");
-            };
+            }
+        }
+
+        // Compute and set attestation subnets
+        let mut config = config.clone();
+        let subnets = compute_subscribed_subnets(enr.node_id(), config.current_epoch);
```

Comment:
if you want for the current epoch you could use NetworkState.status.head_slot/32(the const of course)

Reply:
done, now i am passing `head_slot ` as an argument to `Discovery::new`